docs/TUTORIAL.md

```diff
@@ -349,79 +175,16 @@ local pod = kubeUtil.app.v1.pod;
 }
 ```
 
-Running something like `jsonnet examples/tutorial/simplePod.jsonnet -m
-.` will produce an `nginxPod.json` containing the JSON below; notice
-that this JSON is a normal, well-formed [v1.Pod][v1-pod] object, which
-uses the container object from above, and which can be passed to
-`kubectl` to run on an arbitrary kubernetes cluster.
+Save the file as pod.libsonnet, and run the following commands 
+to deploy the pod to your cluster:
 
-```json
-{
-    "apiVersion": "v1",
-    "kind": "Pod",
-    "meadata": {
-        "labels": {
-          "app": "nginx"
-        }
-    },
-    "spec": {
-        "containers": [
-          {
-              "image": "nginx:1.7.9",
-              "imagePullPolicy": "Always",
-              "livenessProbe": {
-                "httpGet": {
-                    "path": "/",
-                    "port": 80
-                },
-                "initialDelaySeconds": 15,
-                "timeoutSeconds": 1
-              },
-              "name": "nginx",
-              "ports": [
-                {
-                    "containerPort": 80,
-                    "name": "http"
-                }
-              ]
-          }
-        ]
-    }
-}
+```bash
+jsonnet pod.libsonnet // create pod.json
+kubectl apply -f pod.json
 ```
 
-## 2b: Simple, static schema validation
-
-Correct-by-default, etc.
-
-## 2c: Flexible customization via mixins
-
-# 3: `kubeUtil.libsonnet`
-
-[intro here]
-
-## 3a: Gitlab: A real-world example
-
-
-[v1]: https://kubernetes.io/docs/api-reference/v1/definitions/ "Kubernetes v1 API"
-[v1beta1]: https://kubernetes.io/docs/api-reference/extensions/v1beta1/definitions/ "Kubernetes v1beta1 API"
-[v1-container]: https://kubernetes.io/docs/api-reference/v1/definitions/#_v1_container "v1.Container"
-[v1-pod]: https://kubernetes.io/docs/api-reference/v1/definitions/#_v1_pod "v1.Pod"
+## Work with mixins
```

Comment:
@Bradamant3 So far, so good but definitely looking forward to seeing this section soon as it should help demonstrate a lot of the power of `ksonnet`